---
layout: post
title: ARTIC Network provides protocol for rapid, accurate sequencing of novel coronavirus (nCoV-2019)
description: first genomes released
image: /images/nanopore-announcement.png
show_tile: false
permalink: news-ncov
category: menu
date: 12.02.2020
---

### When used with nanopore sequencing technology, the workflow can go from sample to genome within 8 hours.

On 5th February, the first nCoV-2019 genome was released, sequenced using methods developed by the ARTIC network for rapid characterisation of the virus.  The ARTIC network, funded by a Collaborative Award by the Wellcome Trust, has developed a set of lab and bioinformatics protocols for nCoV-2019 that enable characterisation of the virus on nanopore sequencers within 8 hours.

Scientists from Hangzhou CDC in China first used the methods developed by ARTIC to generate a high accuracy nCoV genome.  The ARTIC method is currently being adopted by other labs in China, and has also been used in labs across four continents.

Dr. Jun Li; Hangzhou Centre for Disease and Control, said:

>“Hangzhou CDC took the lead in completing the nation's first 2019-nCov genome assembly using only nanopore data. No other technology is required to correct the data. The final assembly result is 100% consistent with the reference genome. With the continuous development of the epidemic, it is possible to carry out accurate real-time monitoring of virus mutations in close proximity to samples and obtain effective information in a short period of time. It is particularly important to understand the development and spread of the epidemic, guide prevention and control work, and vaccine research."

Professor Nicholas Loman; University of Birmingham and ARTIC principal investigator, said:

>“This work has been built off our experience in developing rapid in-field sequencing solutions for viral epidemics. By producing viral sequences quickly and sharing this data on an open source platform, it is possible to maximise the utility of genomics data as a component of epidemic response.”

This first ARTIC-generated genome is now publicly available on GISAID, for use in international efforts to understand the epidemiology of the outbreak.  The workflow package developed by ARTIC includes a targeted multiplex primer scheme, real-time analysis using the RAMPART software package, and full step-by-step instructions: from sample to genome sequence ready for sharing with the international community.

The rapid provision of an optimised nCoV sequencing protocol was possible because of the experience of the ARTIC group in previous outbreak situations, including the West African Ebola virus epidemic of 2013-2016.  For the past three years, the ARTIC project has been developing an end-to-end system for processing samples from viral outbreaks to generate real-time epidemiological information that is interpretable and actionable by public health bodies.  The team, made up of scientists from the universities of Edinburgh, Birmingham, Cambridge, Oxford, KU Leuven, UCLA and the Fred Hutchinson Cancer Centre, has previously developed methods, delivered ‘lab in a suitcase’ sequencing resources/solutions and collaborative local training in multiple outbreaks including Ebola, Zika virus and Yellow Fever. They are currently involved in projects that work with local scientists to provide support to the ongoing Ebola outbreak in DRC, poliovirus surveillance in Pakistan and measles surveillance in Rwanda.

ARTIC has collaborated with Oxford Nanopore, whose sequencing technology delivers data in real-time, contributing to the rapid 8 hour end to end workflow.  In January, Oxford Nanopore noted that it had sent 200 of their portable MinION devices to China, to enable the rapid, local sequencing of nCoV in a broad network of public health laboratories.

## The impact of genome sequence data in an outbreak

Viral genome data generated prospectively during outbreaks can help provide information about relatedness to other viruses, mode and tempo of evolution, geographical spread and adaptation to human hosts. This information can be used to assist in epidemiological investigations, particularly when combined with other types of data (e.g. case counts).  Faster data generation and sharing can contribute to a better public health response.

Rapid availability of sequence data can also support vaccine development and the development of diagnostic capabilities.

## Nanopore sequencing 

Oxford Nanopore has developed a range of devices are available for nanopore sequencing, from the portable MinION to the large scale PromethION.  The devices sequence DNA or RNA in real-time, offering rapid insights.  Nanopores process any length fragment, and so can sequence very long reads if required.  The technology has been used in multiple outbreak situations including communicable diseases, food pathogens, and specifically in the understanding of drug resistant pathogens.  Beyond microbiology, nanopore sequencing is also used in many other areas of scientific research including human genetics, cancer research, plant and environmental applications.

## Data sharing in an outbreak situation: acknowledgements

These resources were only possible because of the genome sequence (accession MN908947) released by The Shanghai Public Health Clinical Center & School of Public Health, in collaboration with the Central Hospital of Wuhan, Huazhong University of Science and Technology, the Wuhan Center for Disease Control and Prevention, the National Institute for Communicable Disease Control and Prevention, Chinese Center for Disease Control, and the University of Sydney, Sydney, Australia.

## About ARTIC

Fast evolving RNA viruses (such as Ebola, MERS, SARS, influenza etc) continually accumulate changes in their genomes that can be used to reconstruct the epidemiological processes that drive the epidemic. The ARTIC is an international network of academics and public health researchers creating a ‘lab-in-a-suitcase’ that can be deployed to remote and resource-limited locations. Targeting a wide-range of emerging viral diseases, the sequence generation – based around a recently developed, single-molecule portable sequencing instrument, the Oxford Nanopore Technology MinION – is closely linked to the analysis platform, integrating associated epidemiological knowledge to reveal the processes of transmission, virus evolution and epidemiological linkage with extremely rapid turn-around. This real-time approach will provide actionable epidemiological insights within days of samples being taken from patients.

## Resources:

* [ARTIC network nCoV-19 website](https://artic.network/ncov-2019), find protocol and other resources
* Nanopore Community [nCoV-2019 research channel](https://community.nanoporetech.com/posts/channels/ncov-2019-research-communi) (login required)
* [Nanopore web page for enquiries](https://register.nanoporetech.com/ryi-novel-coronavirus)

*Oxford Nanopore products are for Research Use Only.*

![img](assets/images/nanopore-announcement.png)